components/main/MainContent.vue

```diff
@@ -63,3 +68,19 @@ const containerClass = computed(() => {
     </div>
   </div>
 </template>
+
+<style>
+  .beta {
+    width: 56.16px;
+    height: 35.16px;
```

Comment:
Why the partial pixels?

Reply:
I took these from Figma... and I'm assuming I'm supposed to round these. I'll make that change and will do so moving forward. I was trying to do the least amount of work and thought you guys would raise it in PR comments if it should be rounded :grimacing: